server/pages/desktop.jade

```diff
@@ -26,6 +26,23 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class='is-desktop' + ( isFluidWidth ?
 		script(src='/calypso/build-desktop.js')
 		script(src='/desktop/desktop-app.js')
 
+		script.
```

Comment:
I had a concern there myself, the `script. \n IIFE` is the format we use in `index.jade`
I wanted this to be in 'shell' because it is not related to any logical part of the app.
But as @johngodley has pointed out, it may be better to move this to `public_desktop/desktop-app.js`